.circleci/config.yml

```diff
@@ -471,19 +397,15 @@ commands:
                 command: circleci-agent step halt
 
 jobs:
-  build-and-deploy-job:
+  app-deploy:
```

Comment:
For me, I like clear and short job names. `app-deploy` is easier to remember than `build-and-deploy-job`.

Also, setting new naming convention for jobs. Since command names starts with a verb like `build` or `run`, to make job names obviously different, job names start with a noun.